app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -218,8 +220,8 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:enabled="@{viewModel.validPin ? true : false}"
```

Comment:
I think this is not needed any more

Reply:
My bad, I thought it is a clickable

Reply:
Done.

Reply:
No, its correct. Earlier I thought it will affect the screen reader but checked it and it works correctly.

Reply:
Thanks for the update, I was about to verify this locally